.circleci/config.yml

```diff
@@ -603,65 +597,23 @@ jobs:
       - image: memcached
     working_directory: /go/src/github.com/grafana/grafana
     steps:
-        - checkout
-        - run:
-            name: ci job started
-            command: './scripts/ci-job-started.sh'
-        - run: dockerize -wait tcp://127.0.0.1:11211 -timeout 120s
-        - run: dockerize -wait tcp://127.0.0.1:6379 -timeout 120s
-        - run:
-            name: cache server tests
-            command: './scripts/circle-test-cache-servers.sh'
-        - run:
-            name: ci job failed
-            command: './scripts/ci-job-failed.sh'
-            when: on_fail
-        - run:
-            name: ci job succeeded
-            command: './scripts/ci-job-succeeded.sh'
-            when: on_success
-
-  end-to-end-test-release:
-      docker:
-        - image: circleci/node:12-browsers
-        - image: grafana/grafana-dev:$CIRCLE_TAG
-      steps:
-          - run: dockerize -wait tcp://127.0.0.1:3000 -timeout 120s
-          - checkout
```

Comment:
this was not used anywhere so removed it

Reply:
@torkelo I and @xlson are discussing how to merge eventual differences between this and end-to-end-tests back in for the release pipeline. I only noticed in retrospect that I had missed that the old release pipeline had a different end-to-end test job. It's also possible for us to dig back in Git history though to find this job again.

Reply:
I do not think we need it

Reply:
@torkelo That would be for the best :)